Orange/widgets/classify/owsaveclassifier.py

```diff
@@ -22,7 +22,9 @@ class OWSaveClassifier(widget.OWWidget):
     #: A list of recent filenames.
     history = Setting([])
 
-    FILTER = "Pickle files (*.pickle *.pck)\nAll files (*.*)"
+
+    SUPPORTED_EXTENSIONS = ('.pkcls', '.pickle', '.pkl')
```

Comment:
There's a thing called backward compatibility. Otherwise, I'm perfectly happy with preventing users from loading their once reliably stored trained classifiers, so .pkcls it is ...